style_rwd.css

```diff
@@ -2,21 +2,233 @@
+        margin: 0;
+    }
+    .margin_menu {
+        margin: 0;
+    }
+    .header_wrapper>div {
+        padding: 5px 0 5px 10px;
+    }
+    .header_wrapper{
+        max-width:767px;
+        max-height:50px;
+    }
+    .section_wrapper{
+        max-width:767px;
+        margin:0 auto;
+        padding: 0;
+    }
+    .font_size{
+        font-size:10px;
+    }
+    .section_hero {
+        padding-bottom: 42px;
     }
     .column{
+        max-width:767px;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
-        font-weight: 30;
-        max-width: 767px;
+        font-size: 40px;
+        margin-bottom: 0;
+        max-width:767px;
+    }
+    .button_container {
+        justify-content: flex-start;
+        max-width:767px;
+        margin: 0 0 0 100px;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
+        padding: 5px 16px;
         font-size: 12px;
     }
-}
+    .section_container {
+        min-height: 300px;
+        justify-content: center;
+    }
+    .catch_phrase {
+        flex-direction: column;
+        max-width:300px;
+    }
 
+    .flex_column {
+        flex-direction: column;
+    }
+
+    .section_heading {
+        font-size: 18px;
+        max-width:767px;
+        padding: 17px 0 20px;
+    }
+    .pros_photo{
+        height: auto;
+        margin: 0 auto;
+    }
+    .pros_container {
+        max-width:767px;
+        margin-top: 15px;
+    }
+    .text {
+        font-size: 16px;
+        margin-bottom: 16px;
+        max-width: 767px;
+    }
+    .mountains_pic{
+        display:none;
+    }
+    .paragraph_description{
+    margin: 0 0 5px 20px;
+    }
+    .longer_paragraph{
+        max-width: 767px;
+        margin-left: 8px;
+    }
+    .max_width{
+        justify-content: center;
+        align-items: flex-start;
+        max-width: 767px;
+    }
+    .section_center_media{
+        justify-content:flex-start;
+    }
+    .right_side{
+        max-width: 767px;
+        height: auto;
+    }
+    .post_container{
+        text-align: center;
+    }
+    .constant_width{
+        max-width: 300px;
+    }
+    .left_side{
+        margin: 40px 0 0 0;
+        display: block;
+    }
+    .pink{
+        margin: 0;
+        display: block;
+        }
+    .left_side_margin{
+        line-height: 25px;
+    }
+    .invite_phrase{
+        line-height:55px;
+    }
+    .button_book_margin{
+        margin: 0 0 0 70px;
+    }
+    .pic{
+        max-width: 100%;
+    }
+
+    .section_pink_wrapper{
+        padding:0 8px;
+    }
+    .paragraph_header {
+        margin-bottom: 8px;
+        margin-top: 16px;
+    }
+    .pros_con_right {
+        padding:0;
+    }
+    .third_page_header{
+        margin-left: 8px;
+    }
+    .padding{
+        flex-direction: column;
+    }
+    .girl_pic{
+        max-width: 252px;
+        max-height: 302px;
+    }
```

Comment:
This image looks wrong on 425px, do you remember what I've told you about the size of responsive images?

Reply:
I am lil bit confused about this cause i found some different sources how do handle pics... for example some says that I should have resized pics for each size and use command like: <source srcset="square-large.jpg 2x,
                square-small.jpg 1x" />
                
                some says that I should use it like this:
                
                <source media="(min-width: 36em)"
   srcset="large.jpg  1024w,
      medium.jpg 640w,
      small.jpg  320w"
   sizes="33.3vw" />
   
   and well I dont know how to exactly handle with pics cause I thought this one will be alot easier if I just use half of dimensions of bigger version.

Reply:
fixed this one